distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1552,6 +1571,130 @@ public void testIngestPlugin() throws Exception {
+        final String symbolicName = "catalog-plugin-metacardingest-network";
+        // ** IP rule broken - not getting what we expect **
+        final String clientIP = "0:0:0:0:0:0:0:1";
+        final String clientScheme = "https";
+
+        Map<String, Object> networkRule1Properties = getServiceManager().getMetatypeDefaults(
+                symbolicName,
+                factoryPid);
+
+        Map<String, Object> networkRule2Properties = Maps.newHashMap(networkRule1Properties);
+
+        List<Object> attributeAdjustmentsForRule1 = ImmutableList.builder()
+                .add("security.access-groups = FIFA, NFL")
+                .build();
+
+        List<Object> attributeAdjustmentsForRule2 = ImmutableList.builder()
+                .add("description = None")
+                .add("metacard-tags = ALPHA, BRAVO, resource")
+                // ** Remove below addition when IP gets fixed **
+                .add("security.access-groups = FIFA, NFL")
+                .build();
+
+        networkRule1Properties.put("criteriaKey", "remoteAddr");
+        networkRule1Properties.put("expectedValue", clientIP);
+        networkRule1Properties.put("newAttributes", attributeAdjustmentsForRule1);
+
+        networkRule2Properties.put("criteriaKey", "scheme");
+        networkRule2Properties.put("expectedValue", clientScheme);
+        networkRule2Properties.put("newAttributes", attributeAdjustmentsForRule2);
+
+        getServiceManager().createManagedService(factoryPid, networkRule1Properties);
+        getServiceManager().createManagedService(factoryPid, networkRule2Properties);
+
+        getServiceManager().waitForRequiredBundles(symbolicName);
+
+        // ** INGEST **
+        // ** Working with two metacard xml files and one txt file to serve as a tika product **
+        String simpleProductId = ingestMetacardAndGetId("simple-product.txt", MediaType.TEXT_PLAIN);
+        String card1Id = ingestMetacardAndGetId("modified-metacard-1.xml", MediaType.TEXT_XML);
+        String card2Id = ingestMetacardAndGetId("modified-metacard-2.xml", MediaType.TEXT_XML);
+
+        // ** VALIDATION **
+        // ** Note only the first validation gets security since it's parsed by tika **
+        startValidateBody(simpleProductId).assertThat()
+                .body(hasNewDescription)
+                .body(hasNewMetacardTags)
+                .body(hasNewSecurityAccessGroups);
+        startValidateBody(card1Id).assertThat()
+                .body(hasInitialDescription)
+                .body(hasNewMetacardTags);
+        startValidateBody(card2Id).assertThat()
+                .body(hasNewDescription)
+                .body(hasInitialMetacardTags);
+
+        // ** CLEAN UP **
+        // ** Don't let the conditions spill-over and impact other tests **
+        getServiceManager().stopFeature(true, "catalog-metacardingest-network");
+
+    }
+
+    private ValidatableResponse startValidateBody(String metacardId) {
```

Comment:
This method could be re-used. Could we make the name a little more generic, e.g., `getMetacardValidatableResponse(String source, String metacardId)`, so people know they can re-use it?